dotcom-rendering/src/web/components/Card/components/CardLink.tsx

```diff
@@ -12,19 +12,30 @@ const fauxLinkStyles = css`
 	left: 0;
 	background-color: transparent;
 
+	opacity: 0;
```

Comment:
setting `opacity: 0` here hides the focus styling for the card.. do you think it could be applied to an inner `<span>` containing the text instead, or something like that?

Reply:
> setting `opacity: 0` here hides the focus styling for the card.. do you think it could be applied to an inner `<span>` containing the text instead, or something like that?

Hm I hadn't noticed that, good catch!

Reply:
Seems like just using `aria-label` is enough to get rid of the error